examples/naive/aarch64/intt_dilithium_1234_5678_manual_ld4.s

```diff
@@ -0,0 +1,512 @@
+        vmls \a, \tmp2, modulus
+.endm
+
+.macro gs_butterfly a, b, root, idx0, idx1
+        vsub     tmp,    \a, \b
+        vadd     \a,    \a, \b
+        mulmodq  \b, tmp, \root, \idx0, \idx1
+.endm
+
+.macro mulmod_v dst, src, const, const_twisted
+        vmul        \dst,  \src, \const
+        vqrdmulh    \src,  \src, \const_twisted
+        vmls        \dst,  \src, modulus
+.endm
+
+.macro gs_butterfly_v a, b, root, root_twisted
+        vsub    tmp,    \a, \b
+        vadd    \a,    \a, \b
+        mulmod  \b, tmp, \root, \root_twisted
+.endm
+
+.macro mul_ninv dst0, dst1, dst2, dst3, dst4, dst5, dst6, dst7, src0, src1, src2, src3, src4, src5, src6, src7
+        mulmod \dst0, \src0, ninv, ninv_tw
+        mulmod \dst1, \src1, ninv, ninv_tw
+        mulmod \dst2, \src2, ninv, ninv_tw
+        mulmod \dst3, \src3, ninv, ninv_tw
+        mulmod \dst4, \src4, ninv, ninv_tw
+        mulmod \dst5, \src5, ninv, ninv_tw
+        mulmod \dst6, \src6, ninv, ninv_tw
+        mulmod \dst7, \src7, ninv, ninv_tw
+.endm
+
+.macro load_roots_1234 r_ptr
+        ldr_vi root0, \r_ptr, (8*16)
+        ldr_vo root1, \r_ptr, (-8*16 + 1*16)
+        ldr_vo root2, \r_ptr, (-8*16 + 2*16)
+        ldr_vo root3, \r_ptr, (-8*16 + 3*16)
+        ldr_vo root4, \r_ptr, (-8*16 + 4*16)
+        ldr_vo root5, \r_ptr, (-8*16 + 5*16)
+        ldr_vo root6, \r_ptr, (-8*16 + 6*16)
+        ldr_vo root7, \r_ptr, (-8*16 + 7*16)
+.endm
+
+.macro load_next_roots_56 root0, r_ptr0
+        ldr_vi \root0, \r_ptr0, 16
+.endm
+
+.macro load_next_roots_6 root0, r_ptr0
+        ldr_vi \root0, \r_ptr0, 8
+.endm
+
+.macro load_next_roots_78 root0, root0_tw, root1, root1_tw, root2, root2_tw, r_ptr1
+        ldr_vi \root0,    \r_ptr1, (6*16)
+        ldr_vo \root0_tw, \r_ptr1, (-6*16 + 1*16)
+        ldr_vo \root1,    \r_ptr1, (-6*16 + 2*16)
+        ldr_vo \root1_tw, \r_ptr1, (-6*16 + 3*16)
+        ldr_vo \root2,    \r_ptr1, (-6*16 + 4*16)
+        ldr_vo \root2_tw, \r_ptr1, (-6*16 + 5*16)
+.endm
+
+.macro transpose4 data
+        trn1_s t0, \data\()0, \data\()1
+        trn2_s t1, \data\()0, \data\()1
+        trn1_s t2, \data\()2, \data\()3
+        trn2_s t3, \data\()2, \data\()3
+
+        trn2_d \data\()2, t0, t2
+        trn2_d \data\()3, t1, t3
+        trn1_d \data\()0, t0, t2
+        trn1_d \data\()1, t1, t3
+.endm
+
+.macro save_gprs // slothy:no-unfold
+        sub sp, sp, #(16*6)
+        stp x19, x20, [sp, #16*0]
+        stp x19, x20, [sp, #16*0]
+        stp x21, x22, [sp, #16*1]
+        stp x23, x24, [sp, #16*2]
+        stp x25, x26, [sp, #16*3]
+        stp x27, x28, [sp, #16*4]
+        str x29, [sp, #16*5]
+.endm
+
+.macro restore_gprs // slothy:no-unfold
+        ldp x19, x20, [sp, #16*0]
+        ldp x21, x22, [sp, #16*1]
+        ldp x23, x24, [sp, #16*2]
+        ldp x25, x26, [sp, #16*3]
+        ldp x27, x28, [sp, #16*4]
+        ldr x29, [sp, #16*5]
+        add sp, sp, #(16*6)
+.endm
+
+.macro save_vregs // slothy:no-unfold
```

Comment:
Note that those still use the old tag notation, and should be replaced by `// @slothy: ...`